eZ/Publish/Core/Limitation/ContentTypeLimitationType.php

```diff
@@ -168,8 +168,8 @@ public function evaluate(APILimitationValue $value, APIUserReference $currentUse
     public function getCriterion(APILimitationValue $value, APIUserReference $currentUser)
     {
         if (empty($value->limitationValues)) {
-            // no limitation values
-            throw new \RuntimeException('$value->limitationValues is empty, it should not have been stored in the first place');
+            // A Policy should not have empty limitationValues stored
+            throw new \RuntimeException('$value->limitationValues is empty');
```

Comment:
Hmm, what is the reasoning behind this change? The message provided some information for a maintainer, which when moved to an inline comment is gone for him.

Reply:
Discussed this with Mikołaj, @mikadamczyk, can you comment?

Reply:
The argument here is that information for a maintainer should go to logs. In the previous form, it is too "technical" for non-maintainer users.

Reply:
> The argument here is that information for a maintainer should go to logs. In the previous form, it is too "technical" for non-maintainer users.

Ah, ok then :+1: